docs/modules/configuring/con-configuring-cluster-connections.adoc

```diff
@@ -0,0 +1,79 @@
+[id='con-configuring-cluster-connections-{context}']
```

Comment:
I don't think the filename or the id are right.   'cluster' is ambiguous because it might refer to the target cluster (the cluster being proxied) or the virtual cluster (if we can improve the terminology great).

con-configuring-cluster-connections-{context} > con-configuring-target-cluster-connections-{context}